web/src/components/AppStoreBanner.tsx

```diff
@@ -108,15 +104,15 @@ const useAppStoreBannerState = (): AppStoreBannerState => {
 };
 
 const getAppStoreUrl = (appStoreIsDismissed: boolean): AppStoreURLs | undefined => {
-  if (appStoreIsDismissed || !isMobileWeb()) {
+  if (appStoreIsDismissed || Capacitor.isNativePlatform()) {
     return undefined;
   }
 
-  if (isAndroid()) {
+  if (/Android|android/.test(navigator.userAgent)) {
     return AppStoreURLs.GOOGLE;
   }
 
-  if (isIphone()) {
+  if (/iPad|iPhone|iPod/.test(navigator.userAgent)) {
```

Comment:
FYI the apple app store is also used on Macs but I don't think it's a situation where we need support that at the moment.